doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
 
 Motivations
 ===========
 
-The current state-of-the-art software solution for speeding up deep learning 
-computation is to integrate kernel libraries like Intel® Math Kernel Library 
-for Deep Neural Networks (Intel® MKL DNN) and Nvidia\*'s CuDNN into deep 
-learning frameworks. These kernel libraries offer a runtime performance boost 
-on specific hardware targets through highly-optimized kernels and other 
-operator-level optimizations.
+The current state-of-the-art software solution for speeding up deep learning computation is to integrate kernel libraries such as Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL DNN) and Nvidia\*'s CuDNN into deep learning frameworks. These kernel libraries offer a performance boost during runtime on specific hardware targets through highly-optimized kernels and other operator-level optimizations.
 
 However, kernel libraries have three main problems: 
 
 #. Kernel libraries do not support graph-level optimizations.
 #. Framework integration of kernel libraries does not scale.
 #. There are too many kernels to write, and they require expert knowledge.
 
-The nGraph Compiler stack is designed to address the first two problems. nGraph 
-applies graph-level optimizations by taking the computational graph from a deep 
-learning framework like TensorFlow\* and reconstructing it with the nGraph 
-:abbr:`Intermediate Representation (IR)`. The nGraph IR centralizes computational 
-graphs from various frameworks and provides a unified way to connect backends 
-for targeted hardware. From here, PlaidML or one of the nGraph transformers can 
-generate code in various forms, including LLVM, OpenCL, OpenGL, Cuda and Metal. 
-This generated code is where the low-level optimizations are automatically 
-applied.  The result is a more efficient execution that does not require any 
-manual kernel integration work for most hardware targets. 
-
-What follows here is more detail about how our solution addresses these 
-problems. 
+The nGraph Compiler stack is designed to address the first two problems. nGraph applies graph-level optimizations by taking the computational graph from a deep learning framework such as TensorFlow\* and reconstructs it with nGraph 
+:abbr:`Intermediate Representation (IR)`. nGraph IR centralizes computational graphs from various frameworks and provides a unified way to connect backends for targeted hardware. To address the third problem, nGraph is integrated with PlaidML, *a lorem ipsum*, to generate code in LLVM, OpenCL, OpenGL, Cuda, and Metal. Low-level optimizations are automatically applied to the generated code, resulting in a more efficient execution that does not require manual kernel integration for most hardware targets. 
 
+The following three sections explore the main problems of kernel libraries in more detail and describe how nGraph addresses them.
 
-Problem: Absence of graph-level optimizations
+Problem 1: Absence of graph-level optimizations
```

Comment:
It may be better to say that problem 1 is that "kernel libraries do not have graph-level optimization".  It's a little redundant but I think it is more clear.